src/shared/module-util.c

```diff
@@ -0,0 +1,64 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <errno.h>
+
+#include "module-util.h"
+
+int module_load(struct kmod_ctx *ctx, const char *module) {
```

Comment:
probably should actually be called "module_load_and_warn()" or so, since it logs all over the place, always.

Reply:
ignore this, i see now that you change that later on